INSTALL.md

```diff
@@ -75,6 +75,14 @@ This list is incomplete as there are too many
 Add `antigen bundle zsh-users/zsh-syntax-highlighting` as the last bundle in
 your `.zshrc`.
 
+#### [Fig](https://fig.io)
+
+Fig adds apps, shortcuts, and autocomplete to your existing terminal.
+
+Install Zsh Syntax Highlighting in just one click.
+
+<a href="https://fig.io/plugins/other/zsh-syntax-highlighting" target="_blank"><img src="https://fig.io/badges/install-with-fig.svg" /></a>
```

Comment:
I'd rather remove the `target="_blank"` (it's the user's decision, not ours).

Please add alt text to the image, unless the image is removed entirely.